package.json

```diff
@@ -89,5 +83,8 @@
     "commitizen": {
       "path": "./node_modules/cz-conventional-changelog"
     }
+  },
+  "exports": {
+    "chainTypes": "./src/chainTypes/index.ts"
```

Comment:
I followed as subquery suggested - https://university.subquery.network/build/manifest.html#working-example.

When I was directly specifying `src/chainTypes/index.ts` it was giving the following lint error - 
```
String does not match the pattern of "^\./".
```

I also got the same errors, but that is just because your build was not properly created. Since we have `incremental: true` specified in `tsconfig.json` the build is not properly updated. Can you remove your `tsconfig.tsbuildinfo` file and try again? That did the trick for me. 

Please let me know if this doesn't works

Reply:
Yeah removing tsconfig.tsbuildinfo worked for me.

We might need to put it into a docker ignore file. Its not the first time it messed me up and I forgot about the fix.